scripts/convert_llama_to_dcp.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import json
+from pathlib import Path
+
+import torch
+import torch.distributed.checkpoint as DCP
+
+from torchtitan.logging import init_logger, logger
+
```

Comment:
Import the function to precompute the `freqs_cis` key.

```suggestion
from torchtitan.models.llama.model import precompute_freqs_cis
```